scraper/avecmondoc/avecmondoc.py

```diff
@@ -339,6 +341,28 @@ def fetch(self, request, client):
             return None
         return first_availability.isoformat()
 
+def get_vaccine_dose(motive_name):
```

Comment:
Si on veut faire les choses bien :
```suggestion
def get_vaccine_dose(motive_name: str) -> Optional[list]:
```